.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/ST-DDT/SpongeAPI.git
```

Comment:
To make it easier you can do it like I did with my Selectors PR, I kept the code in one commit and the API ref/module changes in another commit. This way the team can manipulate it easier.